.github/workflows/ci.yml

```diff
@@ -16,25 +16,42 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  get_version:
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.get_version.outputs.version }}
+
+    steps:
+      - name: Check out pipeline code
+        uses: actions/checkout@v3
+
+      - name: Get Minimum supported Nextflow version
+        id: get_version
+        run: |
+          VERSION=$(grep 'nextflowVersion' nextflow.config | awk -F'= ' '{print $2}' | tr -d '!>=" ')
+          echo "version=$VERSION" >> $GITHUB_OUTPUT
```

Comment:
Does this need to be a whole separate job? It's just going to add extra jobs to the runner and code checkouts.